test/extensions/geoip_providers/maxmind/geoip_provider_test.cc

```diff
@@ -98,6 +108,34 @@ TEST_F(GeoipProviderTest, ValidConfigCityAndIspDbsSuccessfulLookup) {
   expectStats("isp_db");
 }
 
+TEST_F(GeoipProviderTest, ValidConfigCityAndIspDbsSuccessfulLookupWithoutResultDontCauseCrash) {
```

Comment:
could you elaborate more/add explanatory comment how this test relates to newly added country mapping feature? E.g. why could there be a crash